.env.template

```
@@ -57,3 +57,9 @@ export NOTIFICATION_DATABASE_URL="postgresql://user:pass@host/notifications"
 # Firebase credentials for notification-service
 # These credentials are baked into the image to be used by the application
 export NOTIFICATION_FIREBASE_CREDENTIALS='{"format":"minified JSON";"see":"https://github.com/PermanentOrg/notification-service/"}'
+
+# New Relic credentials for Performance monitoring
+# These are configured on GitHub; they are listed here only for the sake of completeness
+# and can be left blank
+export NEW_RELIC_APPLICATION_NAME_DEV=Backend-dev
```

I just noticed that the variable name has a `DEV` on the end; we don't want to specify the name of the environment in the names of the variables.
```suggestion
export NEW_RELIC_APPLICATION_NAME=Backend-dev
```